dist/idf-mobilite.js

```diff
@@ -133,7 +133,8 @@ class IDFMobiliteCard extends LitElement {
                     if (lineRef != "") {
                         const nextDeparture = Math.floor((new Date(Date.parse(stop.MonitoredVehicleJourney.MonitoredCall.ExpectedDepartureTime)) - Date.now()) / 1000 / 60)
                         const lineStop = stop.MonitoredVehicleJourney.DestinationRef.value.substring(stop.MonitoredVehicleJourney.DestinationRef.value.indexOf(":Q:") + 3, stop.MonitoredVehicleJourney.DestinationRef.value.length - 1)
-                        if ((!exclude_lines || exclude_lines.indexOf(lineRef) == -1) && (!exclude_lines_ref || exclude_lines_ref.indexOf(lineStop) == -1) && nextDeparture > -5 && nextDeparture < 60) {
+                        const mission = stop.MonitoredVehicleJourney.JourneyNote != "" ? stop.MonitoredVehicleJourney.JourneyNote[0].value : ""
+                        if ((!exclude_lines || exclude_lines.indexOf(lineRef) == -1) && (!exclude_lines_ref || exclude_lines_ref.indexOf(lineStop) == -1) && (!exclude_lines_missions || exclude_lines_missions.indexOf(mission) == -1) && nextDeparture > -5 && nextDeparture < 60) {
```

Comment:
Maybe it would be better to create a `filter()` method?  In the future it will check all filter values and return true/false